cluster/images/conformance/Makefile

```diff
@@ -54,9 +54,14 @@ endif
 	chmod a+rx ${TEMP_DIR}/e2e.test
 	chmod a+rx ${TEMP_DIR}/gorunner
 
-	cd ${TEMP_DIR} && sed -i.back "s|ARCH|${ARCH}|g; s|BASEIMAGE|${BASEIMAGE}|g" Dockerfile
+	cd ${TEMP_DIR} && sed -i.back "s|BASEIMAGE|${BASEIMAGE}|g" Dockerfile
 
-	docker build --pull -t ${REGISTRY}/conformance-${ARCH}:${VERSION} ${TEMP_DIR}
+	docker buildx build \
```

Comment:
- where is buildx being initialized in the case that the host does not have it initialized already?
- we're not setting the DOCKER_CLI_EXPERIMENTAL=enabled anywhere that I see either?

Reply:
> * where is buildx being initialized in the case that the host does not have it initialized already?

Currently nowhere, we have an `init-buildx.sh` script in k/release which could be vendored over here: https://github.com/kubernetes/release/blob/master/hack/init-buildx.sh

Do we wanna do that?

> * we're not setting the DOCKER_CLI_EXPERIMENTAL=enabled anywhere that I see either?

Same as above, the CI / release process right now ensures that it is set. Following docker versions will set it per default as well: https://github.com/docker/cli/blob/e31e00585363e6a989e37fa92cf06481ea218344/docs/deprecated.md#configuration-options-for-experimental-cli-features

Anyways, we can still add it to the build scripts, which should not hurt.

Reply:
@saschagrunert Related: looks like we are not setting it currently in the build canary (https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-build-canary/1355584153801723904)

Do you want to go ahead and handle setup in k/k or try to just fix jobs in intermediate?

Reply:
> Currently nowhere, we have an init-buildx.sh script in k/release which could be vendored over here: https://github.com/kubernetes/release/blob/master/hack/init-buildx.sh

well I'd say yes, but the original source for that is in fact:
https://github.com/kubernetes-sigs/kind/commits/master/hack/build/init-buildx.sh
🙃 

Without this, you can't consistently rely on there being a builder provisioned, causing builds to fail on clean hosts (except on docker desktop, where there's one provided OOTB).